CI/run_reframe.sh

```diff
@@ -50,11 +50,14 @@ fi
 if [ -z "${EESSI_TESTSUITE_BRANCH}" ]; then
     EESSI_TESTSUITE_BRANCH='v0.4.0'
 fi
-if [ -z "${EESSI_CVMFS_REPO}" ]; then
-    export EESSI_CVMFS_REPO=/cvmfs/software.eessi.io
-fi
-if [ -z "${EESSI_VERSION}" ]; then
-    export EESSI_VERSION=2023.06
+if [ -z "${USE_EESSI_SOFTWARE_STACK}" ] | [ $USE_EESSI_SOFTWARE_STACK == "True" ]; then
+    export USE_EESSI_SOFTWARE_STACK=True
```

Comment:
It would be good to print some more of these variables in the section that also prints reframe config etc. Makes things easier to debug.

Reply:
I've now added these two
```
echo "Using EESSI: ${USE_EESSI_SOFTWARE_STACK}"
echo "MODULEPATH: ${MODULEPATH}"
```